dashboard/test/ui/features/dance/social_share.feature

```diff
@@ -0,0 +1,62 @@
+# Testing that the social share buttons are shown/hidden in the correct scenarios
+# in Dance Party.
+
+Feature: Social share buttons
+  Scenario: Signed in 13+ user sees social share buttons
+    Given I create a teacher named "Hermoine"
+    And I am on "http://studio.code.org/s/dance/stage/1/puzzle/13"
+    And I rotate to landscape
+    And I wait for the page to fully load
+
+    And I wait until I see selector ".project_share"
+    When I click ".project_share"
+    And I wait for 10 seconds
+    Then element "#project-share" is visible
+    Then element ".social-buttons .fa-facebook" is visible
+    And element ".social-buttons .fa-twitter" is visible
+
+    And I sign out
+
+  Scenario: Signed in user under 13 does not see social share buttons
+    Given I create a young student named "Harry"
+    And I am on "http://studio.code.org/s/dance/stage/1/puzzle/13"
+    And I rotate to landscape
+    And I wait for the page to fully load
+
+    And I wait until I see selector ".project_share"
+    When I click ".project_share"
+    And I wait for 10 seconds
```

Comment:
Should this be changed to 3 seconds like the other two scenarios?

I'd normally suggest we wait for something other than an set time, but in this case with the reachability check that runs I'm not sure what we'd wait for.

Reply:
😱 i was testing something and never changed it back to 3 seconds!